src/tools/tidy/src/features.rs

```diff
@@ -89,6 +89,13 @@ pub fn check(
             &src_path.join("test/ui"),
             &src_path.join("test/ui-fulldeps"),
             &src_path.join("test/compile-fail"),
+            // This allows `tidy` to pick up features that are `rustdoc`-specific.
+            //
+            // This way, `tidy` is happy because all features have a test file
+            // and all the features are correctly tested (either through `rustdoc`
+            // or `rustc`), while avoiding having  to run `rustdoc` for all other
```

Comment:
nit: there's an extra space here

```suggestion
            // or `rustc`), while avoiding having to run `rustdoc` for all other
```